ajax/libs/jQuery.Gantt/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "jQuery.Gantt",
+  "version": "1.2.2",
+  "description": "jQuery Gantt Chart is a simple chart that implements gantt functionality as a jQuery component.",
+  "filename": "js/jquery.fn.gantt.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mbielanczuk/jQuery.Gantt.git"
+  },
+  "keywords": [
+    "jquery",
+    "gantt",
+    "chart"
+  ],
+  "author": {
+    "name": "Marek Bielańczuk",
+    "url": "https://github.com/mbielanczuk"
+  },
+  "licenses": [
+    "MIT",
+    "GPL-2.0"
+  ],
+  "homepage": "https://mbielanczuk.com/jquery-gantt/",
```

Comment:
Maybe just use `http` is better. What do you think?

The SSL certification is a little bit weird. as  https://www.ssllabs.com/ssltest/analyze.html?d=mbielanczuk.com
The website use Let's Encrypt Authority, and there are 2 certificates.
One of the certificates is not Trusted.

So the `https://mbielanczuk.com` show not security in chrome.